rfcs/015 - Reworked Compiler Front End.md

```diff
@@ -0,0 +1,183 @@
+# 000 - RFC Template
```

Comment:
Can this be fixed and switched to `016 - Reworked Compiler Front End RFC` and the file moved to `016 - Reworked Compiler Front End RFC.md`